src/RamIndexAnalysis.h

```diff
@@ -134,29 +134,28 @@ class MaxMatching {
 };
 
 /**
- * @class IndexSet
+ * @class MinIndexSelection
  * @brief computes the minimal index cover for a relation
- *
- * This class is a helper class for RamIndexAnalysis.
+ *        in a RAM Program.
  *
  * If the indexes of a relation can cover several searches, the minimal
  * set of indexes is computed by Dilworth's problem. See
  *
- * "Optimal On The Fly Index Selection in Polynomial Time"
- * https://arxiv.org/abs/1709.03685
+ * "Automatic Index Selection for Large-Scale Datalog Computation"
+ * http://www.vldb.org/pvldb/vol12/p141-subotic.pdf
  *
  */
 
-class IndexSet {
+class MinIndexSelection {
 public:
-    using LexicographicalOrder = std::vector<int>;
-    using OrderCollection = std::vector<LexicographicalOrder>;
+    using LexOrder = std::vector<int>;
```

Comment:
Is there an advantage to shortening the descriptive name? It seems less informative now.

Reply:
LexOrder is a known term in CS.